apps/web/src/components/milestone-logs/recruitment/MilestoneView.tsx

```diff
@@ -46,8 +46,11 @@ export const MilestoneView = ({ milestone, children }: MilestoneViewProps) => {
           )}
           {children}
         </div>
-        <span className='text-sm text-black break-words block py-1'>
-          {setOutcomeText(milestone.status.status)}
+        <span
+          className='text-sm text-bcBlueLink font-bold break-words block py-1'
+          style={{ fontStyle: 'italic' }}
```

Comment:
I don't know why but tailwind `italic` is not working.

Reply:
could be the custom font were using, I only see bold and regular in @font-face